zetta_utils/mazepa_layer_processing/common/blendable_apply_flow.py

```diff
@@ -69,51 +295,105 @@ def make_tasks_without_blending(
 
         return tasks
 
-    def make_tasks_with_blending(
+    def make_tasks_with_checkerboarding(  # pylint: disable=too-many-locals
         self,
         idx: VolumetricIndex,
-        idx_chunks: Iterable[VolumetricIndex],
+        red_chunks: Iterable[VolumetricIndex],
         dst: VolumetricLayer,
         **kwargs: P.kwargs,
-    ) -> Tuple[List[mazepa.tasks.Task[R_co]], List[List[VolumetricLayer]]]:
-        """Makes tasks that will generate tasks that can be reduced to the final output,
-        alongside the list of the temporary layers for each chunk in idx_chunks that need to be
-        reduced for the final output in that chunk."""
-        if self.temp_layers_dir is None:
-            raise ValueError("`temp_layers_dir` must be specified when using blending")
-        if self.blend_pad is None:
-            raise ValueError("`blend_pad` must be specified when using blending")
-
-        blending_weighter = VolumetricDataBlendingWeighter(self.blend_pad, self.blend_mode)
-
-        tasks = []
-        idx_chunks_temps: List[List[VolumetricLayer]] = [[] for _ in idx_chunks]
-        for chunker, chunker_idx in self.chunker.split_into_nonoverlapping_chunkers(
+    ) -> Tuple[
+        List[mazepa.tasks.Task[R_co]],
+        List[mazepa.tasks.Task[None]],
+        List[List[VolumetricIndex]],
+        List[List[VolumetricLayer]],
+        List[List[VolumetricLayer]],
```

Comment:
This feels like a lot of of lists, hard to understand which is which.
Would it be better to declare a small class up above, with 5 named fields, and return an instance of that class?

Reply:
I don't think the overhead (both for the person having to dig through to find the class and from a performance standpoint) is worth it given that it's only used twice; I think a class might be necessary when arbitrary blending is implemented, but for now I'd say no.